.github/workflows/ecs_deploy.yaml

```
@@ -88,16 +95,16 @@ jobs:
 
-      - name: Change task definition ETH
-        id: change_task_definition_eth
+      - name: Change task definition for ${{ matrix.network }}
+        id: change_task_definition
         env:
           VAR_ENV_NAME: ${{ secrets[format('{0}_ENV_NAME', env.PREFIX)] }}
-          VAR_APP_NAME: "bundler-eth"
+          VAR_APP_NAME: "bundler-${{ matrix.network }}"
           VAR_IMAGE: ${{ secrets[format('{0}_ECR_URL', env.PREFIX)] }}/${{ secrets[format('{0}_AWS_REPOSITORY', env.PREFIX)] }}:${{ github.sha }}
-          VAR_ERC4337_BUNDLER_ETH_CLIENT_URL: ${{ secrets[format('{0}_ERC4337_BUNDLER_ETH_CLIENT_URL', env.PREFIX)] }}
+          VAR_ERC4337_BUNDLER_ETH_CLIENT_URL: ${{ secrets[format('{0}_ERC4337_BUNDLER_{1}_CLIENT_URL', env.PREFIX, toupper(matrix.network))] }}
```

actionlint

undefined function "toupper". available functions are "always", "cancelled", "contains", "endswith", "failure", "format", "fromjson", "hashfiles", "join", "startswith", "success", "tojson"

(expression)